doc/man7/ossl-guide-tls-server-block.pod

```diff
@@ -0,0 +1,329 @@
+example).
+
+    /*
+     * Load the server's certificate *chain* file (PEM format), which includes
+     * not only the leaf (end-entity) server certificate, but also any
+     * intermediate issuer-CA certificates.  The leaf certificate must be the
+     * first certificate in the file.
+     *
+     * In advanced use-cases this can be called multiple times, once per public
+     * key algorithm for which the server has a corresponding certificate.
+     * However, the corresponding private key (see below) must be loaded first,
+     * *before* moving on to the next chain file.
+     */
+    if (SSL_CTX_use_certificate_chain_file(ctx, "chain.pem") <= 0) {
+        SSL_CTX_free(ctx);
+        ERR_print_errors_fp(stderr);
+        errx(res, "Failed to load the server certificate chain file");
+    }
+
+    /*
+     * Load the corresponding private key, this also checks that the private
+     * key matches the just loaded end-entity certificate.  It does not check
+     * whether the certificate chain is valid, the certificates could be
+     * expired, or may otherwise fail to form a chain that a client can validate.
+     */
+    if (SSL_CTX_use_PrivateKey_file(ctx, "pkey.pem", SSL_FILETYPE_PEM) <= 0) {
+        SSL_CTX_free(ctx);
+        ERR_print_errors_fp(stderr);
+        errx(res, "Error loading the server private key file, "
+                  "possible key/cert mismatch???");
+    }
+
+Next we enable session caching, which makes it possible for clients to more
+efficiently make additional TLS connections after completing an initial full
+TLS handshake.  With TLS 1.3, session resumption typically still performs a fresh
+key agreement, but the certificate exchange is avoided.
+
+    /*
+     * Servers that want to enable session resumption must specify a cache id
+     * byte array, that identifies the server application, and reduces the
+     * chance of inappropriate cache sharing.
+     */
+    SSL_CTX_set_session_id_context(ctx, (void *)cache_id, sizeof(cache_id));
+    SSL_CTX_set_session_cache_mode(ctx, SSL_SESS_CACHE_SERVER);
+
+    /*
+     * How many client TLS sessions to cache.  The default is
+     * SSL_SESSION_CACHE_MAX_SIZE_DEFAULT (20k in recent OpenSSL versions),
+     * which may be too small or too large.
+     */
+    SSL_CTX_sess_set_cache_size(ctx, 1024);
+
+    /*
+     * Sessions older than this are considered a cache miss even if still in
+     * the cache.  The default is two hours.  Busy servers whose clients make
+     * many connetions in a short burst may want a shorter timeout, on lightly
+     * loaded servers with sporadic connections from any given client, a longer
+     * time may be appropriate.
+     */
+    SSL_CTX_set_timeout(ctx, 3600);
+
+Most servers, including this one, do not solicit client certificates.  We
+therefore do not need a "trust store" and allow the handshake to complete even
+when the client does not present a certificate.  [ Even if a client did present a
```

Comment:
Not sure how the square braces will render with a pod file. I don't we use them anywhere else. Are they really needed? Consider just including the text as-is without `[]`

Reply:
Done.